content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -173,8 +173,7 @@ Cluster Domain will be set to `cluster.local` unless
 
 ### Stable Storage
 
-Kubernetes creates one [PersistentVolume](/docs/concepts/storage/persistent-volumes/) for each
-VolumeClaimTemplate. In the nginx example above, each Pod will receive a single PersistentVolume
+Per each StatefulSet triggered pod Kubernetes creates a PersistentVolumeClaim object for each VolumeClaimTemplates entry defined in the StatefulSet object.In the nginx example above, each Pod will receive a single PersistentVolume
```

Comment:
- `pod` -> `Pod`
- Added line breaks
  - (As-is: There is no space between `object.` and `In`)

```suggestion
Per each StatefulSet triggered Pod Kubernetes creates a PersistentVolumeClaim object 
for each VolumeClaimTemplates entry defined in the StatefulSet object.
In the nginx example above, each Pod will receive a single PersistentVolume
```

Reply:
@jihoon-seo Thanks for the review. I will be updating this

Reply:
@jihoon-seo Addressed the Changes. PTAL!

Reply:
Hi @jihoon-seo all the objectives for the required changes are addressed, Please let me know for any further changes if required.
Thanks